.eslintignore

```diff
@@ -2,5 +2,5 @@
 dist/
 types/
 /test/restDocs/*.json
-/test/restDocs/results/
-/test/restDocs/snapshots/
+/test/restDocs/__tests__/
```

Comment:
`__tests__` folder [by default](https://jestjs.io/docs/configuration#testmatch-arraystring) is used to find `jest` tests.
But in our case, it looks like it's used to store the generation results, so I got a bit confused by this. The name should better represent the contents of the folder: temporary results of the generator used for tests. This would be a very long name, so probably we can shorten it to "test-results" or "temp-results"?

Reply:
Or should it rollback to the original `results`?

Reply:
Yeah, results are good enough, thanks!